app/src/main/java/com/kamron/pogoiv/Pokefly.java

```diff
@@ -806,7 +806,7 @@ private void populatePrevScanNarrowing(IVScanResult ivScanResult) {
      */
     private void populateResultsHeader(IVScanResult ivScanResult) {
         resultsPokemonName.setText(ivScanResult.pokemon.name);
-        resultsPokemonLevel.setText(getString(R.string.level) + ": " + ivScanResult.estimatedPokemonLevel);
+        resultsPokemonLevel.setText(getString(R.string.level) + " " + ivScanResult.estimatedPokemonLevel);
```

Comment:
@oskomonzon Can you weigh in on this?

I also think this should be a format string (because that's always the right thing for translateable strings).